### DISCUSSION TITLE
convert html into corresponding to_markdown

### DISCUSSION THREAD

**Discussion by ManzerHa-Kore** [markedjs/marked#2153] [Q&A]
![imgg](https://user-images.githubusercontent.com/50398944/127596921-ce3791d2-2ef8-4445-8787-3b19b87b8ec4.png)

I am trying to convert  "{panel:bgColor=#deebff}\n* {code:java}let x= 1;let z= x+5;console.log(\"Hello ,\",z){code}\n{panel}" into proper markdown, so that when markdown passed to marked(valid_Panel_markdown) converts a valid html to display the UI as it is in image.

What should be valid markdown for 'panel'.
marked(valid_Panel_markdown) gives  desired html

Is marked.js have some vice-versa method to get valid markdown format when html passed.

**Comment by UziTech** [comment]
marked only converts markdown to html not the other way around. [Turndown](https://www.npmjs.com/package/turndown) is an npm package that converts html into markdown.

There is no "panel" element in markdown but since valid HTML is also valid markdown I think the correct markdown for that would be:

````markdown
<panel bgColor="deebff">

```java
let x= 1;let z= x+5;console.log("Hello ,",z)
```

</panel>
````
